index.js

```diff
@@ -537,7 +537,7 @@
   /**
    * Handle "click" events.
    */
-
+  var hasProcess = new Function('return typeof process')() !== 'undefined';
```

Comment:
Why does this need to be evaled?

Reply:
When a `process` variable is included in code directly, some packing tools like Webpack will auto [polyfill](https://webpack.js.org/configuration/node/#node-process) a fake "process" object into the browser environment. So It cannot be detected to "undefined" correctly. But if it is in the string of `new Function` parameter, packing tools won't detect it and provide no polyfills.

Reply:
Ah! I see. We might be switching to rollup (https://github.com/visionmedia/page.js/issues/444) in which case it might not pack the process polyfill anyways. I'll keep an eye on that when I work on that issue.